.github/workflows/test-js-client.yml

```diff
@@ -40,7 +40,7 @@ jobs:
 
       - name: Test
         working-directory: ./clients/js
-        run: pnpm test
+        run: pnpm test -- --match='!*das:*'
```

Comment:
Done. https://github.com/metaplex-foundation/mpl-core/pull/30/commits/66251a9f5dc6c39de57db5fba30ec10b2b6ca539

I don't like doing it via the AVA config though, because to make the tests work locally we'll have to constantly mess around with the `files` field there. Ideally, that same glob pattern should be added via CLI in GH actions only, to make the tests work locally by default but ignore them by CI. Unfortunately, I couldn't find a way do that yet. That very same glob pattern is not recognized by CLI for some reason.

Reply:
Used `match` with a different glob pattern. https://github.com/metaplex-foundation/mpl-core/pull/30/commits/126c1768c7dc0eb22dc33b484c39e02112c0cc16